wren/internal/materialization_client.go

```diff
@@ -201,7 +200,7 @@ func (c *materializationClient) GetAllMaterializationsBySourceType(ctx context.C
 		Str("source_type", string(sourceType)).
 		Logger()
 
-	materializations, err := c.entClient.Materialization.Query().Where(materialization.SourceTypeEQ(sourceType)).All(ctx)
+	materializations, err := c.entClient.Materialization.Query().Where(materialization.SourceTypeEQ(sourceType)).WithOwner().All(ctx)
```

Comment:
panic occurred because materialization objects weren't pre-populated with the Owner object.